test/integration/patients/sidebar/action-sidebar.js

```diff
@@ -742,6 +748,110 @@ context('action sidebar', function() {
     cy.clock().invoke('restore');
   });
 
+  specify('action attachments', function() {
+    const actionData = {
+      id: '1',
+      attributes: {
+        name: 'Name',
+        details: 'Details',
+        duration: 5,
+        due_date: testDateSubtract(2),
+        due_time: null,
+        updated_at: testTs(),
+      },
+      relationships: {
+        owner: { data: null },
+        state: { data: { id: '22222' } },
+        files: { data: [{ id: '1' }, { id: '2' }] },
+      },
+    };
+
+    cy
+      .routeTeams()
+      .routeGroupsBootstrap()
+      .routeAction(fx => {
+        fx.data = actionData;
+
+        fx.data.relationships.owner.data = {
+          id: '11111',
+          type: 'clinicians',
+        };
+
+        return fx;
+      })
+      .routePatientActions()
+      .routePatientFlows()
+      .routeActionActivity()
+      .routeActionComments()
+      .routeActionFiles(fx => {
+        fx.data = [
+          {
+            id: '1',
+            attributes: {
+              path: 'https://www.bucket_name.s3.amazonaws.com/HRA_v1.pdf',
+              created_at: '2019-08-24T14:15:22Z',
+            },
+          },
+          {
+            id: '2',
+            attributes: {
+              path: 'https://www.bucket_name.s3.amazonaws.com/HRA_v2.pdf',
+              created_at: '2019-08-25T14:15:22Z',
+            },
+          },
+        ];
+
+        return fx;
+      })
+      .routePrograms()
+      .routeAllProgramActions()
+      .routeAllProgramFlows()
+      .routePatient()
+      .visit('/patient/1/action/1')
+      .wait('@routePatientActions')
+      .wait('@routePatientFlows')
+      .wait('@routeAction')
+      .wait('@routeActionActivity')
+      .wait('@routeActionComments')
+      .wait('@routePatient');
+
+    cy
+      .get('.sidebar')
+      .find('[data-attachments-files-region]')
+      .children()
+      .children()
+      .as('attachmentItems')
+      .should('have.length', 2);
+
+    cy
+      .get('@attachmentItems')
+      .first()
+      .find('.action-sidebar__attachment-filename')
+      .should('contain', 'HRA_v2.pdf')
+      .should('have.attr', 'href')
+      .and('contain', 'https://www.bucket_name.s3.amazonaws.com/HRA_v2.pdf');
+
+    cy
+      .get('@attachmentItems')
+      .first()
+      .find('.action-sidebar__attachment-filename')
+      .should('have.attr', 'target')
+      .and('contain', '_blank');
+
+    cy
+      .get('@attachmentItems')
+      .first()
+      .find('.action-sidebar__attachment-download')
+      .should('have.attr', 'href')
+      .and('contain', 'https://www.bucket_name.s3.amazonaws.com/HRA_v2.pdf');
+
+    cy
+      .get('@attachmentItems')
+      .first()
+      .find('.action-sidebar__attachment-download')
+      .should('have.attr', 'download');
```

Comment:
I'm simply testing the attributes/properies of the `<a></a>` for both the filename and download button.

* Filename: that it has the correct `href` and the `target=_blank` to open the file in a new tab.
* Download button: that it has the correct `href` and the `download` attribute to download the file to the users device.

Does this work? Or do we need to actually open the links in the tests?